submodules/faxbox/faxbox.js

```diff
@@ -233,6 +261,14 @@ define(function(require) {
 
 				delete results.current_user;
 
+				var invalidCallerID = self.getCallerID(results);
```

Comment:
This method can be replaced by:
```suggestion
				var invalidCallerID = _.find(results.phone_numbers, _.get(results.faxbox, 'caller_id', null));
```
You can add a comment to explain what it does but it should be pretty straightforward to read.